modules/openapi-generator/src/main/resources/typescript/api_doc.mustache

```diff
@@ -22,27 +22,25 @@ Method | HTTP request | Description
 
 
 ```typescript
-import { {{{moduleName}}} } from '{{{projectName}}}';
-import * as fs from 'fs';
+import {{{moduleName}}} from '{{{projectName}}}';
 
 const configuration = {{{moduleName}}}.createConfiguration();
 const apiInstance = new {{{moduleName}}}.{{classname}}(configuration);
 
 {{#hasParams}}
-let body:{{{moduleName}}}.{{classname}}{{operationIdCamelCase}}Request = {
+const request: {{{moduleName}}}.{{classname}}{{operationIdCamelCase}}Request = {
 {{#allParams}}
-  // {{{dataType}}}{{#description}} | {{{description}}}{{/description}}{{^required}} (optional){{/required}}
+  {{#description}}  // {{{description}}}{{^required}} (optional){{/required}}{{/description}}
   {{paramName}}: {{{example}}},
 {{/allParams}}
 };
 {{/hasParams}}
 {{^hasParams}}
-let body:any = {};
+const request = {};
 {{/hasParams}}
 
-apiInstance.{{{operationId}}}(body).then((data:any) => {
-  console.log('API called successfully. Returned data: ' + data);
-}).catch((error:any) => console.error(error));
+const data = await apiInstance.{{{operationId}}}(request);
```

Comment:
changed this to `await`, then we don't need to do an error handling example and the code gets a bit shorter